config/rbac/submariner-globalnet/cluster_role.yaml

```diff
@@ -5,15 +5,6 @@ metadata:
   creationTimestamp: null
   name: submariner-globalnet
 rules:
-  - apiGroups:
```

Comment:
To resovle the chicken and egg scenarion, how about we don't delete this RBAC yet and just add it for gateways. This will allow submariner PR to pass as well as find other issues, if any. Then we can create another PR to delete nodes RABC.